### PR TITLE
Replace deprecated colcon_core.task.check_call() with colcon_core.task.run()

### PR DESCRIPTION
**Pull request by ivanpauno**
`check_call()` was deprecated and renamed `run()`, see also https://github.com/colcon/colcon-core/pull/334.

### CODE REVIEW

**Comment by ivanpauno** [comment]
@jacobperron could you review?

I don't have write access here, so feel free to merge when you think this is ready.

**Review by jacobperron — APPROVED** [review]
LGTM

Before merging, it would be nice to have green CI so I've opened https://github.com/colcon/colcon-gradle/pull/29.

**Comment by jacobperron** [comment]
@ivanpauno Could you rebase this change? Hopefully CI will go green after that :crossed_fingers:

**Comment by ivanpauno** [comment]
> @ivanpauno Could you rebase this change? Hopefully CI will go green after that

Done, CI is passing now

**Review by jacobperron — APPROVED** [review]

**Pull request merged**
